src/sentry/web/frontend/debug/mail.py

```diff
@@ -208,6 +210,19 @@ def make_performance_event(project):
     return perf_event
 
 
+def make_generic_event(project):
+    occurrence, group_info = process_event_and_issue_occurrence(
+        TEST_ISSUE_OCCURRENCE.to_dict(),
+        {
+            "event_id": uuid.uuid4().hex,
+            "project_id": project.id,
+            "timestamp": before_now(minutes=1).isoformat(),
+        },
+    )
+    generic_group = group_info.group
+    return generic_group.get_latest_event()
```

Comment:
If it's helpful we could probably return the event from `process_event_and_issue_occurrence` as well